scripts/quests/sandoria/Her_Majesty_S_Garden.lua

```diff
@@ -0,0 +1,103 @@
+-----------------------------------
+-- Her Majestys Garden
+-----------------------------------
+-- Log ID: 0, Quest ID: 62
+-- Chalvatot : !gotoid 17731598
+-- Map of the Northlands
+-----------------------------------
+require('scripts/globals/npc_util')
+require('scripts/globals/items')
+require('scripts/globals/keyitems')
+require('scripts/globals/quests')
+require('scripts/globals/status')
+require('scripts/globals/zone')
+require('scripts/globals/interaction/quest')
+-----------------------------------
+local ID = require("scripts/zones/Chateau_dOraguille/IDs")
+
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.SANDORIA, xi.quest.id.sandoria.HER_MAJESTY_S_GARDEN)
+
+quest.reward =
+{
+    fame     = 30,
+    fameArea = xi.quest.fame_area.SANDORIA,
+    item     = xi.ki.MAP_OF_THE_NORTHLANDS_AREA,
```

Comment:
Shouldn't this be keyitem?

Reply:
Don't know if it matters? It gave me the KI - but for consistency sake changing.